### DISCUSSION TITLE
Custom-Boot-Display-K1Max

### DISCUSSION THREAD

**Discussion by IS78CO** [Guilouz/Creality-Helper-Script-Wiki#484] [Issues]
Hallo ich habe nach Anleitung https://github.com/KORSiRO/Custom-Boot-Display-K1-K1Max
versucht das umzusetzen, nur lässt mich bei der SSH Verbindung im Root Verzeichnis, die Originalen Dateien  nicht Runterladen.
Mache das wie beschrieben.

was mache ich falsch. ??
![Screenshot 2024-03-25 162941](https://github.com/Guilouz/Creality-K1-Series/assets/94009691/9823ec24-41c3-4e24-a670-892dc451aa5b)

**Comment by Guilouz** [comment]
In english please.
